cmd/operator-sdk/bundle/cmd.go

```diff
@@ -20,13 +20,13 @@ import (
 
 //nolint:structcheck
 type bundleCmd struct {
-	directory      string
-	packageName    string
+	Directory      string
+	PackageName    string
 	imageTag       string
-	imageBuilder   string
-	defaultChannel string
+	ImageBuilder   string
+	DefaultChannel string
 	channels       string
-	generateOnly   bool
+	GenerateOnly   bool
```

Comment:
why we need these changes?

Reply:
I had to make a bunch of fields public to be able to set them in the tests. It's because cobra normally populated the fields set via flag when you initialize the command struct, but I can't do it that way because I'm not cobra. So I have to make them public so I can manually set them.